base/applications/calc/resource.h

```diff
@@ -1,126 +1,128 @@
-#define IDC_BUTTON_STA        1009
-#define IDC_BUTTON_AVE        1010
-#define IDC_BUTTON_SUM        1011
-#define IDC_BUTTON_S          1012
-#define IDC_BUTTON_DAT        1013
-#define IDC_BUTTON_FE         1014
-#define IDC_BUTTON_DMS        1015
-#define IDC_BUTTON_SIN        1016
-#define IDC_BUTTON_COS        1017
-#define IDC_BUTTON_TAN        1018
-#define IDC_BUTTON_LEFTPAR    1019
-#define IDC_BUTTON_EXP        1020
-#define IDC_BUTTON_XeY        1021
-#define IDC_BUTTON_Xe2        1022
-#define IDC_BUTTON_Xe3        1023
-#define IDC_BUTTON_RIGHTPAR   1024
-#define IDC_BUTTON_LN         1025
-#define IDC_BUTTON_LOG        1026
-#define IDC_BUTTON_NF         1027
-#define IDC_BUTTON_RX         1028
-#define IDC_BUTTON_MC         1029
-#define IDC_BUTTON_MR         1030
-#define IDC_BUTTON_MS         1031
-#define IDC_BUTTON_MP         1032
-#define IDC_BUTTON_PI         1033
-#define IDC_BUTTON_7          1034
-#define IDC_BUTTON_4          1035
-#define IDC_BUTTON_1          1036
-#define IDC_BUTTON_0          1037
-#define IDC_BUTTON_A          1038
-#define IDC_BUTTON_8          1039
-#define IDC_BUTTON_5          1040
-#define IDC_BUTTON_2          1041
-#define IDC_BUTTON_SIGN       1042
-#define IDC_BUTTON_B          1043
-#define IDC_BUTTON_9          1044
-#define IDC_BUTTON_6          1045
-#define IDC_BUTTON_3          1046
-#define IDC_BUTTON_DOT        1047
-#define IDC_BUTTON_C          1048
-#define IDC_BUTTON_DIV        1049
-#define IDC_BUTTON_MULT       1050
-#define IDC_BUTTON_SUB        1051
-#define IDC_BUTTON_ADD        1052
-#define IDC_BUTTON_D          1053
-#define IDC_BUTTON_MOD        1054
-#define IDC_BUTTON_OR         1055
-#define IDC_BUTTON_LSH        1056
-#define IDC_BUTTON_EQU        1057
-#define IDC_BUTTON_E          1058
-#define IDC_BUTTON_AND        1059
-#define IDC_BUTTON_XOR        1060
-#define IDC_BUTTON_NOT        1061
-#define IDC_BUTTON_INT        1062
-#define IDC_BUTTON_F          1063
-#define IDC_RADIO_QWORD       1064
-#define IDC_RADIO_DWORD       1065
-#define IDC_RADIO_WORD        1066
-#define IDC_RADIO_BYTE        1067
-#define IDC_RADIO_DEG         1068
-#define IDC_RADIO_RAD         1069
-#define IDC_RADIO_GRAD        1070
-#define IDC_BUTTON_CANC       1071
-#define IDC_BUTTON_CE         1072
-#define IDC_BUTTON_BACK       1073
-#define IDC_TEXT_OUTPUT       1074
-#define IDC_TEXT_PARENT       1075
-#define IDC_TEXT_MEMORY       1076
-#define IDC_LIST_STAT         1079
-#define IDC_BUTTON_RET        1080
-#define IDC_BUTTON_LOAD       1081
-#define IDC_BUTTON_CD         1082
-#define IDC_BUTTON_CAD        1083
-#define IDC_TEXT_NITEMS       1084
-#define IDC_BUTTON_SQRT       1085
-#define IDC_BUTTON_PERCENT    1086
-#define IDC_BUTTON_CONVERT    1087
-#define IDC_BUTTON_FOCUS      1089
-#define IDC_COMBO_CATEGORY    1090
-#define IDC_COMBO_FROM        1091
-#define IDC_COMBO_TO          1092
-#define IDC_BUTTON_RSH        1093
+#define IDS_CALC_NAME                   1
```

Comment:
`#pragma once` missing?

Reply:
It does not seem to me that this file is included more than one time in the sources.